src/app/pages/CreateConnectorPage/CreateConnectorPage.tsx

```diff
@@ -59,26 +60,28 @@ export const CreateConnectorPage: FunctionComponent<
           type={'default'}
           variant={'light'}
         >
-          <CreateConnectorWizard
-            header={<ConnectorWizardHeader />}
-            onClose={openLeaveConfirm}
-          />
-          <Modal
-            title={t('leaveCreateConnectorConfirmModalTitle')}
-            variant={'small'}
-            isOpen={askForLeaveConfirm}
-            onClose={closeLeaveConfirm}
-            actions={[
-              <Button key="confirm" variant="primary" onClick={onClose}>
-                Confirm
-              </Button>,
-              <Button key="cancel" variant="link" onClick={closeLeaveConfirm}>
-                Cancel
-              </Button>,
-            ]}
-          >
-            {t('leaveCreateConnectorConfirmModalDescription')}
-          </Modal>
+          <ConnectorSelectionListCacheStorageProvider>
+            <CreateConnectorWizard
+              header={<ConnectorWizardHeader />}
+              onClose={openLeaveConfirm}
+            />
+            <Modal
+              title={t('leaveCreateConnectorConfirmModalTitle')}
+              variant={'small'}
+              isOpen={askForLeaveConfirm}
+              onClose={closeLeaveConfirm}
+              actions={[
+                <Button key="confirm" variant="primary" onClick={onClose}>
+                  Confirm
```

Comment:
we can use i18n text here

Reply:
yeah, good catch!  Updated that and the other button in here.